src/styles/blocks/starts.scss

```diff
@@ -0,0 +1,36 @@
+.card__stars {
+  display: flex;
+  justify-content: space-between;
+  margin: 17px 16px 25px;
+}
```

Comment:
this belongs to card block. move it to card.scss

Reply:
It is better not to use both margin-top and margin-bottom to set spacing between elements. use one of them. It could cause margin collapse